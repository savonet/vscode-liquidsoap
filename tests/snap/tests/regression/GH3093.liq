def f() =
  x = false

  ignore(x)

#
  x = true

  if x then test.pass() else test.fail () end
end

test.check(f)
