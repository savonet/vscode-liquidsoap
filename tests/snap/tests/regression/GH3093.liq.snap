>def f() =
#^^^ source.liquidsoap keyword.control.liquidsoap
#   ^^^^^^^ source.liquidsoap
>  x = false
#^^^^^^^^^^^^ source.liquidsoap
>
>  ignore(x)
#^^^^^^^^^^^^ source.liquidsoap
>
>#
#^^ source.liquidsoap comment.line.number-sign.liquidsoap
>  x = true
#^^^^^^^^^^^ source.liquidsoap
>
>  if x then test.pass() else test.fail () end
#^^ source.liquidsoap
#  ^^ source.liquidsoap keyword.control.liquidsoap
#    ^^^ source.liquidsoap
#       ^^^^ source.liquidsoap keyword.control.liquidsoap
#           ^^^^^^^^^^^^^ source.liquidsoap
#                        ^^^^ source.liquidsoap keyword.control.liquidsoap
#                            ^^^^^^^^^^^^^^ source.liquidsoap
#                                          ^^^ source.liquidsoap keyword.control.liquidsoap
>end
#^^^ source.liquidsoap keyword.control.liquidsoap
>
>test.check(f)
#^^^^^^^^^^^^^^ source.liquidsoap
>